### DISCUSSION TITLE
Add a console.log() button

### DISCUSSION THREAD

**Discussion by jojojojojoj5564656465465** [lukehaas/RunJS#557] [Ideas]
It would be great on hover a variable or code make a console.log() of this code and also shortcut please

![image](https://user-images.githubusercontent.com/45184918/231441939-034f63a8-bc98-49d2-8caf-71e25973ffed.png)

**Comment by lukehaas** [comment]
An easy way to insert console.log sounds reasonable. However, having a button appear on hover is non-standard. It would be more appropriate to locate this inside a context menu.
